### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by aneallaryea100**
# Mobile Menu
* When the user clicks (or taps) the hamburger button, the mobile menu appears.
* When the user clicks (or taps) the close (X) button, the mobile menu disappears.
* When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
* When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -70,7 +70,39 @@ header nav {
 header nav i {
   color: #fff;
   font-size: larger;
-  margin: px;
+}
+
+.navvme {
```
Comment (Ismailco) [inline style.css:75]: - [ ] Great work implementing the mobile menu, however, the design is different from the Figma one. kindly adjust your design to be like the provided design as much as possible, including the fonts, margin/padding, and overall design

|Your design|Figma design|
|-|-|
|Screenshot_20220620_224015|Screenshot_20220620_224138|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @aneallaryea100, @andym80,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (aneallaryea100) [reply style.css:75]: Design adjusted to be like the Figma design.

File: README.md [hunk 2]
```diff
@@ -23,6 +25,11 @@
 * LinkdIn: [@aneallaryea](https://www.linkedin.com/in/nii-aneal-84ba7a147)
 * Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 
+👤 Andy Menutti
+* Github: [@andym80](https://github.com/andym80)
+* LinkdIn: [@andym80](https://shorturl.at/cADGM)
+
+
```
Comment (tresorsawasawa) [inline README.md:33]: - [ **OPTIONAL** ] Good job :+1: :100:  making all changes required by the previous reviewer. But, I think removing unnecessary code lines would be better to make your documentation code look cleaner.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @aneallaryea100 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @aneallaryea100, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**